### PR TITLE
COMMERCE-12014   Change the default sort order in discount admin page to be based on datemodified desc

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/COMMERCE-12014

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/odata/entity/v2_0/DiscountEntityModel.java [hunk 1]
```diff
@@ -30,7 +32,11 @@ public DiscountEntityModel() {
 					"channelId", locale -> "commerceChannelId")),
 			new CollectionEntityField(
 				new IntegerEntityField(
-					"orderTypeId", locale -> "commerceOrderTypeId")));
+					"orderTypeId", locale -> "commerceOrderTypeId")),
+			new DateTimeEntityField(
+				"dateModified",
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/odata/entity/v2_0/DiscountEntityModel.java:37]: sort

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
just a sort in odata entity. The rest LGTM.

Thank you!

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Review by alessiorendina — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
Re-sent at #4121.

**Pull request closed without merging**